RileyLink/RileyLinkDeviceTableViewCell.swift

```diff
@@ -1,51 +1,13 @@
 //
 //  RileyLinkDeviceTableViewCell.swift
-//  Naterade
+//  RileyLink
```

Comment:
Sorry, this diff got weird because I moved the file out of `RileyLink/` into the framework's directory, then re-added a file in its place which declared `IdentifiableClass` conformance (since that's really just a storyboard convenience, it should stay in the target with the storyboard)